Makefile.am

```diff
@@ -1,56 +0,0 @@
-ACLOCAL_AMFLAGS = -Im4
-
-SUBDIRS = src tests docs
-dist_doc_DATA = README.md
-
-check-style:
-	@lines=`git diff -U0 --no-color --relative origin/main -- ':!src/pkcs11.h' | clang-format-diff -p1 |wc -l`; \
-	if [ "$$lines" != "0" ]; then \
-		echo "Coding Style issues detected"; \
-		exit 1; \
-	else \
-		echo "Coding Styles checks out"; \
-	fi
-
-check-style-show:
-	git diff -U0 --no-color --relative origin/main -- ':!src/pkcs11.h' | clang-format-diff -p1
-
-check-style-fix:
-	git diff -U0 --no-color --relative origin/main -- ':!src/pkcs11.h' | clang-format-diff -i -p1
-
-generate-code:
-	for pfile in src/*.pre; do \
-		gfile=`echo $${pfile} | sed s/\.pre/\.gen\.c/`; \
-		echo "/* DO NOT EDIT, autogenerated from $${pfile} */" > "$${gfile}"; \
-		echo "/* Modify $${pfile} then run make generate-code */" >> "$${gfile}"; \
-		cat $${pfile} | $(CC) -E - | grep -v "^#" > "$${gfile}.tmp"; \
-		sed -i -n -e '/^BEGIN:$$/,$$p' "$${gfile}.tmp"; \
-		sed -i 's/^BEGIN:$$//' "$${gfile}.tmp"; \
-		cat "$${gfile}.tmp" >> $${gfile}; \
-		clang-format -i --verbose "$${gfile}"; \
-		rm "$${gfile}.tmp"; \
-	done
-
-generate-docs:
-	for mdfile in docs/*.md; do \
-		echo "Processing $${mdfile}"; \
-		manfile=`echo $${mdfile} | sed s/\.md//`; \
-		pandoc --standalone --to man $${mdfile} -o $${manfile}; \
-	done
-
```

Comment:
Where are the replacements for check-style, check-style-show, check-stuyle-fix, generate-code and generate-docs ?

Reply:
I think we could add a Makefile for those "maintainer-mode" targets. Let me try in the next iteration.

Reply:
Done.